doc/specs/Authentication.md

```diff
@@ -0,0 +1,473 @@
+# Authentication system
+
+## Metadata
+
+Author: DutchDude
```

Comment:
I'm in favour of keeping it as it unambiguously determines who to slap. Although someone else might contribute changes, these things need to have a sense of ownership.